_uw-research-computing/file-avail-largedata.md

```diff
@@ -157,66 +155,32 @@ back at a later date. Files can be taken off of `/staging` using similar
 mechanisms as uploaded files (as above). 
 
 # 3. Using Staged Files in a Job
+## A. Transferring Large Input Files
+Staged files should be specified in the job submit file using the `osdf://` or `file:///` syntax,
+depending on the size of the files to be transferred. [See this table for more information](htc-job-file-transfer#transferring-data-to-jobs-with-transfer_input_files).
 
-As shown above, the staging directory for large data is `/staging/username`. 
-All interaction with files in this location should occur within your job's 
-main executable.
-
-## A. Accessing Large Input Files
-
-To use large data placed in the `/staging` location, add commands to your 
-job executable that copy input 
-from `/staging` into the working directory of the job. Program should then use 
-files from the working directory, being careful to remove the coiped 
-files from the working
-directory before the completion of the job (so that they're not copied
-back to the submit server as perceived output). 
-
-Example, if executable is a shell script:
+```
+transfer_input_files = osdf://chtc/staging/username/file
```

Comment:
Could we add an example with multiple files - some from home + some from staging?

Reply:
ah, I see this is in the other guide.

Reply:
I'll still add this here!